src/btree/bt_curnext.c

```diff
@@ -644,14 +644,14 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
     WT_DECL_RET;
     WT_PAGE *page;
     WT_SESSION_IMPL *session;
-    size_t pages_skipped_count, total_skipped, skipped;
+    size_t total_skipped, skipped;
     uint32_t flags;
-    bool newpage, restart;
+    bool ignore_tombstone, newpage, restart;
 
     cursor = &cbt->iface;
     session = CUR2S(cbt);
-    pages_skipped_count = 0;
     total_skipped = 0;
+    ignore_tombstone = F_ISSET(&cbt->iface, WT_CURSTD_IGNORE_TOMBSTONE);
```

Comment:
There's a comment problem in this function, search for `not- found` (notice the space).

Reply:
Fixed.